JavaScript/JavaScriptSDK/AppInsights.ts

```diff
@@ -155,14 +155,40 @@ module Microsoft.ApplicationInsights {
          * Starts timing how long the user views a page or other item. Call this when the page opens. 
          * This method doesn't send any telemetry. Call {@link stopTrackTelemetry} to log the page when it closes.
          * @param   name  A string that idenfities this item, unique within this HTML document. Defaults to the document title.
+         * @param startDate A date that identifies the original start date that will override the current date. Defaults to the current date.
          */
-        public startTrackPage(name?: string) {
+        public startTrackPage(startDate?: Date); //Method firm with date
+        public startTrackPage(name?: string, startDate?: Date); //Method firm with name and date
+        public startTrackPage(param1?: (string | Date), param2?: Date) {
```

Comment:
Hi @KamilSzostak We followed your advice and implement the Internal methods. 
Thanks for your feedback.